examples/all_conditions.rs

```diff
@@ -66,6 +67,12 @@ fn spawn(mut commands: Commands) {
                 Tap::new(0.5),
                 bindings![TestTap::KEY],
             ));
+            context.spawn((
+                Action::<TestCooldown>::new(),
+                Cooldown::new(1.0),
+                bindings![TestCooldown::KEY],
+            ));
+
```

Comment:
I'd suggest to either put it last, to shift the assigned keys. So the ordering of 1,2,3...-,= will be consistent with action definition and spawning. It's just easier to navigate this way :slightly_smiling_face: